grouper/fe/handlers.py

```diff
@@ -717,6 +718,136 @@ def get(self, group_id=None, name=None):
         )
 
 
+class AuditsComplete(GrouperHandler):
```

Comment:
Agreed, though the best information there comes from what we tell people on the audit-create template.  We can update the FAQ correspondingly.